sdcm/cluster.py

```diff
@@ -2678,7 +2681,8 @@ def _gen_nodetool_cmd(self, sub_cmd, args, options):
         return f"{self.add_install_prefix('/usr/bin/nodetool')} {options} {sub_cmd} {args}"
 
     def run_nodetool(self, sub_cmd, args="", options="", timeout=None,
-                     ignore_status=False, verbose=True, coredump_on_timeout=False):
+                     ignore_status=False, verbose=True, coredump_on_timeout=False,
+                     decrease_severity_on_exception=None, error_message=""):
```

Comment:
`decrease_severity_on_exception` -> `warning_event_on_exception`
instead of using filter let's have an argument called `publish_event` -> True|False

Reply:
Done
Test passed: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/yulia/job/staging-10gb-3h/166/console
When nodetool is start by cluster health checker, the event is not published